asset/css/search-editor.less

```diff
@@ -18,6 +18,15 @@
     }
   }
 
+  select:not([multiple]) {
+    .appearance(none);
+    padding-right: 1.5625em;
+    background-image: url('@{iplWebAssets}/img/select-icon-white.svg');
```

Comment:
The icon won't load at my machine.

![Screen Shot 2021-09-29 at 14 14 08](https://user-images.githubusercontent.com/6977434/135266662-a46af2d4-da64-4288-8b45-12f4ec81decf.png)

It's technically a `select`. But I'm not sure if we should actually have the icon, since we use the terms design and those don't have it.

Reply:
restart the container for the icon to appear. The terms in the bar are not selects. But they behave like any other input in the bar and every input is auto-completed, hence why it should be okay to not have an icon there.

In the editor the input is not auto completed and doesn't behave as a text input (no cursor on focus), so the user has to know how to interact with this thing, hence the icon.

Reply:
> restart the container for the icon to appear.

Ah, got it to work. The white icon doesn't have enough contrast, though. I would rather use `select-icon.svg`, to also convey the interactivity, but this is difficult as well (remembering the "flicker contrast" thing).

See

![Screen Shot 2021-09-29 at 15 11 59](https://user-images.githubusercontent.com/6977434/135275239-a8571151-7ab4-4f30-b679-60ef4bd3bc0f.png)

So I uploaded a new one (see 65cfd8d) which seems ok.

![Screen Shot 2021-09-29 at 15 13 51](https://user-images.githubusercontent.com/6977434/135275400-17705bcf-f4c8-4b25-b9b3-0f2146337867.png)

> The terms in the bar are not selects. But they behave like any other input in the bar and every input is auto-completed, hence why it should be okay to not have an icon there.
> In the editor the input is not auto completed and doesn't behave as a text input (no cursor on focus), so the user has to know how to interact with this thing, hence the icon.

Agree! Sounds reasonable.

Reply:
I was on v2.9.3 (no dark theme) at the time. With master the white icon has indeed contrast issues. Your's is way better.